frontend/Dockerfile

```diff
@@ -7,10 +7,14 @@ LABEL maintainer OSG Software <help@opensciencegrid.org>
 RUN groupadd -o -g 993 frontend
 RUN useradd -o -u 996 -g 993 -s /bin/sh frontend
 
-RUN yum -y install glideinwms-vofrontend-standalone
-
-RUN yum -y install vo-client \
-    git
+RUN yum clean all && \
+    yum -y --enablerepo=osg-development --enablerepo=osg-upcoming install \
+      git \
+      condor \
+      condor-classads \
+      condor-procd \
+      glideinwms-vofrontend-standalone \
+      vo-client 
```

Comment:
All this is already installed except, of course, glideinwms-vofrontend-standalone

Reply:
I think we should be explicit - there is a good chance we will have to split it up later based on where the software team puts things.